Sources/SnowballSwiftKit/Views/Chip.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  Chip.swift
+//
+//
+//  Created by Vivian Phung on 6/30/23.
+//
+
+import SwiftUI
+import SnowballAssetKit
+
+public enum ChipSize {
+    case small
+    case medium
+}
+
+public struct Chip: View {
+    @Binding var isActive: Bool
+    var isDisabled: Bool
```

Comment:
instead of having an explicit `isDisabled` property we should probably instead use the environment var and isDisabled modifier.

`@Environment(\.isEnabled) var isEnabled`

and you can change it's value up the hierarchy using `isDisabled`.